build/azure-pipelines/build-product.yml

```diff
@@ -51,31 +51,25 @@ stages:
       pool:
         vmImage: $(imageName)
       steps:
-        - task: PowerShell@2
-          displayName: Set last tag to variable
-          name: getVersion
-          inputs:
-            targetType: 'inline'
-            script: |
-              $VERSION_TAG = git describe --tags (git rev-list --tags --max-count=1)
-              Write-Host("##vso[task.setvariable variable=VERSION_TAG]$VERSION_TAG")
-              Write-Host($VERSION_TAG)
+        - template: build-tag.yml
+        - script: |
+            echo $(getVersion.VERSION_TAG)

```

Comment:
>getVersion.VERSION_TAG) [](http://example.com/codeflow?start=19&length=23)

isn't this just $(VERSION_TAG)?

Reply:
The output variables from task of same job are referred using TaskName.VariableName convention as mentioned [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#use-output-variables-from-tasks).